openpype/hosts/blender/api/utils.py

```diff
@@ -446,21 +446,28 @@ def make_paths_absolute(source_filepath: Path = None):
 
         for d in relative_datablocks:
             try:
-                # Check if datablock is a resource file
-                for file in workfile_repre.get("files"):
-                    if Path(file.get("path", "")).name == Path(
-                        d.filepath
-                    ).name:
-                        # Make resource datablock path absolute,
-                        # starting from workdir
-                        d.filepath = str(
-                            Path(
-                                bpy.path.abspath(
-                                    d.filepath,
-                                )
-                            ).resolve()
+                # Check is datablock is a resource file or an UDIM
+                if (
+                    isinstance(d, bpy.types.Image)
+                    and d.source == "TILED"
+                    and not d.packed_file
+                    or any(
+                        (
+                            Path(file.get("path", "")).name
+                            == Path(d.filepath).name
+                            for file in workfile_repre.get("files")
```

Comment:
Isn't it faster like that?
```suggestion
                    or Path(d.filepath).name in {Path(file.get("path", "")).name for file in workfile_repre.get("files")}
```